src/modules/better-table/modules/table-operation-menu.ts

```diff
@@ -227,6 +229,7 @@ export default class TableOperationMenu {
     this.cellBorderWidth = options.border && options.border.widths ? options.border.widths : DEFAULT_BORDER_WIDTH;
     this.cellBorderWidthSubTitle =
       options.border && options.border.text ? options.border.text : DEFAULT_BORDER_WIDTH_SUBTITLE;
+    this.actionSubTitle = options.actions && options.actions.text ? options.actions.text : DEFAULT_ACTION_SUBTITLE;
```

Comment:
teraz je trochu divné že titulky k buttonom nastavujem cez key `items` ale k nadpisu sekcie sa dostanem cez `actions`. Lepší by to bylo mít všechno pod `actions` ale to je spousta refactoringu takže bych spíš tohle `actions` přidal pod `items`